src/javascript/init.js

```diff
@@ -0,0 +1,9 @@
+import {registry} from '@jahia/ui-extender';
+import register from './Administration.register';
+
+export default function () {
+    registry.add('callback', 'jahiaAdministration', {
+        targets: ['jahiaApp-init:1'],
```

Comment:
previously jahiaApp-init was at level 2 now it's level 1? on purpose, mistake?

Reply:
not on purpose, but I don't think it changes anything. i will put it back to 2